src/kademlia/dht_tracker.cpp

```diff
@@ -416,40 +533,52 @@ namespace libtorrent { namespace dht
 		n->list().push_back(entry(node));
 	}
 
+	void save_nodes(entry& ret, node const& dht, std::string const& key)
```

Comment:
neat. free function! it would be nice to have a test for it too, since it should be simple